core/src/main/java/com/netease/arctic/io/reader/ArcticDeleteFilter.java

```diff
@@ -58,11 +64,22 @@
 
 /**
  * Abstract implementation of filtering equality delete files with insert files and base files.
```

Comment:
It seems the class description should be updated

Reply:
ok